src/types/countrySmallProps.ts

```diff
@@ -0,0 +1,24 @@
+export type Flags = {
+  png: string;
+};
+export type Name = {
+  common: string;
+  official: string;
+};
+
+export type Country = {
+  flags: Flags;
+  name: Name;
+  population: number;
+  region: string;
+  capital: string[];
+};
+
+export type CountrySmallProps = {
+  flags: string;
+  name: string;
+  officialName: string;
+  population: number;
+  region: string;
+  capital: string[];
+};
```

Comment:
use definitions to not be necessary export types
countrySmallProps.d.ts. Find the doc about it